### PR TITLE
Sidekiq!

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
#### What? Why?

Twinned with https://github.com/openfoodfoundation/ofn-install/issues/745

Switches from `DelayedJob` to `Sidekiq` for background jobs processing.

#### What should we test?

All jobs; emails, subs, PDF generation, and the HeartbeatJob that tracks the jobs queue status.

#### Release notes

Switch jobs processing from DelayedJob to Sidekiq

Changelog Category: Technical changes

### CODE REVIEW

File: config/routes/admin.rb [hunk 1]
```diff
@@ -1,9 +1,13 @@
+require 'sidekiq/web'
+require 'sidekiq-scheduler/web'
+
 Openfoodnetwork::Application.routes.draw do
   namespace :admin do
 
     authenticated :spree_user, -> user { user.admin? } do
       mount DelayedJobWeb, at: '/delayed_job'
```
Comment (sauloperez) [inline config/routes/admin.rb:8]: we should remove this one now

**Review by sauloperez — COMMENTED** [review]
:fire: :metal:

**Review by sauloperez — APPROVED** [review]

File: config/routes/admin.rb (hunk 1 above)
Comment (Matt-Yorkley) [reply config/routes/admin.rb]: Done.

**Comment by filipefurtad0** [comment]
Hey!

This is working like a charm now. On staging-FR (18) and staging-UK (16) manually tested:

- Order Confirmation
- Order Shipment
- Order Cancellation
- Invoice (per Email)
- Subscription Placement
- Subscription Confirmation
- Shopfront Order (Emails): Confirmation and Cancelling
- Bulk Invoice Printing

Everything works as expected on both servers :tada: 

Emails are delivered much quicker.... :zap: 

I'd say this is good to go from the test cases above. Awesome effort :clap: :clap: :clap:

**Comment by sauloperez** [comment]
I'll wait for the build to finish before merging. I'm so looking forward! :joy: :muscle:

**Comment by Matt-Yorkley** [comment]
FYI the heartbeat job / job queue status can be checked at `/api/v0/status/job_queue` and the Sidekiq jobs dashboard can be seen at `/admin/sidekiq` (when logged in as superadmin).

**Comment by sauloperez** [comment]
that new endpoints means we need to update our HappyApps background jobs check. At least Katuma is currently pointing at `/api/status/job_queue`

**Comment by Matt-Yorkley** [comment]
The previous API namespacing change included a n automatic redirect from `/api/*` to `/api/v0/*` :ok_hand:

**Pull request merged**